Software/backend/database/MongoDB/Main.md

```diff
@@ -0,0 +1,42 @@
+<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
+# MonogoDB
+MongoDB is a cross-platform document-oriented database program. 
+Classified as a NoSQL database program, MongoDB uses JSON-like documents with optional schemas. 
+MongoDB is developed by MongoDB Inc. and licensed under the Server Side Public License (SSPL).
+
+# Table Of Contents
+
+* [Intro](#Intro) 
+* [Platform](#Platform)
+* [Repositories](#Repositories)
+* [Docs](#Docs)
+
+# Intro
+* Document databases differ from relational databases in how they organize data as well as the query languages they support. 
+Document databases are usually categorized under the NoSQL umbrella term.
+
+# Platform
+* The recommended paltfrom to use is [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)
+* [Atlas Guide](./AtlasGuide)
+* [Combine with Node.js](./AtlasWithNode)
+
+# Repositories
```

Comment:
do you mean dependencies? also, maybe add a short explanation as to what these libraries do and why we need them here.